handlers/controller.js

```diff
@@ -23,6 +23,41 @@ let welcome = (req, res) => {
     res.status(StatusCode.EASTER_EGG).send('Welcome to the Greenmap-API!');
 };
 
+
+let getTopManufacturers = async(req, res) => {
+    connectDb().then(async () => {
+        if(!req.params.id) {
+            return res.status(StatusCode.PRECONDITION_FAILED).send(null);
+        }      
+        let barcode = req.params.id;
+        try {
+            let doc = await models.Product.findOne({ barcode });
+            if(doc) {
+                console.log(`found ${doc.barcode} in mongodb`);
+                if(!doc.category || doc.category == null) {
```

Comment:
Do we need to check if it exists and is null? I think we can just check if the `doc.category` is present.